packages/teleport/src/Kubes/AddKube/__snapshots__/AddKube.story.test.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`render failed 1`] = `null`;
```

Comment:
the snapshot results don't look correct (i would always check the snapshot result to make sure it's what we wanted to freeze, sometimes we target the incorrect element)

Reply:
Yes, thanks!